nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -696,6 +694,14 @@ def do_one_task(self, client, num_of_threads, gpu, lock, timeout=60.0, task_name
 
         return stop_run, next_client, end_run_client
 
+    def _get_new_sys_path(self):
+        if not sys.path[0]:
+            new_sys_path = sys.path[1:]
```

Comment:
We want to skip all the sys path of ""?
Could the sys.path be `["", "", "/usr/python3/"]`?

Reply:
Changed the _get_new_sys_path() implementation to exclude all the "" in the sys.path.